### PR TITLE
Support 256 colors in screen terminals

### PR DESCRIPTION
**Pull request by mrtracy**

### CODE REVIEW

File: util/log/clog.go [hunk 1]
```diff
@@ -844,7 +844,7 @@ func (l *loggingT) getTermColorProfile() *colorProfile {
 			case "ansi", "xterm-color":
 				l.colorProfile = colorProfile8
 				color = true
-			case "xterm-256color":
+			case "xterm-256color", "screen-256color":
```
Comment (petermattis) [inline util/log/clog.go]: Is this something where we should be looking for `strings.HasSuffix(term, "-256color")`?

**Comment by petermattis** [comment]
LGTM

File: util/log/clog.go (hunk 1 above)
Comment (mrtracy) [inline util/log/clog.go]: I have no idea honestly, I thought everything used "xterm" at this point, but apparently tmux and screen do not.  The entire thing is confusing.

Comment (bdarnell) [inline util/log/clog.go]: The right way would be to look in the terminfo database, but that means an extra dependency (unless we're picking one up automatically due to our CLI dependencies). Looking for a suffix of "-256color" sounds reasonable to me.

**Comment by bdarnell** [comment]
LGTM

**Pull request merged**